chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -105,6 +105,15 @@
   only_if { cinder_controller && (!use_ha || is_cluster_founder) }
 end
 
+template "/usr/sbin/crowbar-monasca-cleanups.sh" do
```

Comment:
`/usr/local/sbin`

Reply:
Are you sure about this? I know it's wrong, viewed strictly from a FHS point of view, but there is a considerable amount of precedent here in that same file. If we were to use `/usr/local/` here, we should refactor everything which will greatly inflate this pull request. If you really want me to do that, please confirm.

Reply:
yeah, it is consistent with all the other scripts (that are also in the wrong location)